src/main/scala/com/daml/projection/JdbcAction.scala

```diff
@@ -184,6 +184,24 @@ trait Bind[-T] {
     override def set(ps: PreparedStatement, pos: Int, value: U) = {
       Bind.this.set(ps, pos, f(value))
     }
+    override def nullSqlType: Int = Bind.this.nullSqlType
+  }
+
+  /**
+   * The [[java.sql.Types]] code that should be used when invoking `setNull` on `PreparedStatement`. Default in this
+   * trait is set to java.sql.Types.NULL, override if a more specific code is needed.
+   */
+  def nullSqlType: Int = java.sql.Types.NULL
+
+  /**
+   * Return a new Bind with the [[java.sql.Types]] code modified, that should be used when invoking `setNull` on
+   * `PreparedStatement`.
+   */
+  def withSqlType(sqlType: Int): Bind[T] = new Bind[T] {
```

Comment:
Perhaps a better name would be `withNullSqlType`.

Reply:
Thanks.

Reply:
Done.